src/main/scala/treadle/TreadleOptions.scala

```diff
@@ -0,0 +1,282 @@
+// See LICENSE for license details.
+
+package treadle
+
+import firrtl.CircuitState
+import firrtl.annotations.{Annotation, NoTargetAnnotation}
+import firrtl.ir.Circuit
+import firrtl.options.{HasShellOptions, ShellOption, Unserializable}
+import firrtl.stage.FirrtlSourceAnnotation
+import treadle.executable.{ClockInfo, TreadleException}
+
+sealed trait TreadleOption extends Unserializable { this: Annotation => }
+
+/**
+  * Tells treadle to write a vcd file during simulation
+  */
```

Comment:
I've got over 60 of these, I've always used this convention. I don't mind changing but it seems to me like it would just clutter this PR, I'll make a note to look at this in a later style PR